usaspending_api/submissions/management/commands/load_dabs_submission_window_schedule.py

```diff
@@ -100,16 +104,39 @@ def handle(self, *args, **options):
 
         if file_path:
             logger.info("Input file provided. Reading schedule from file.")
-            submission_schedule_objs = self.read_schedules_from_csv(file_path)
+            incoming_schedule_objs = self.read_schedules_from_csv(file_path)
         else:
             logger.info("No input file provided. Generating schedule from broker.")
-            submission_schedule_objs = self.generate_schedules_from_broker()
+            incoming_schedule_objs = self.generate_schedules_from_broker()
+
+        logger.info("Loading existing DABS Submission Window Schedules")
+        existing_schedules = DABSSubmissionWindowSchedule.objects.all()
+
+        # Iterate over each incoming schedule object to find a matching existing
+        # schedule. If there is a match, use the existing submission_reveal_date
+        for incoming_schedule in incoming_schedule_objs:
+            for existing_schedule in existing_schedules:
+                if int(incoming_schedule.id) == existing_schedule.id:
+                    incoming_schedule.submission_reveal_date = existing_schedule.submission_reveal_date
+
+            if incoming_schedule.is_quarter == "True":
```

Comment:
This made me realize that both quarterly and monthly submissions can use the `submission_due_date` field for determining instead of a unique field for each. The CSV is based on prod data, so it's `submission_due_date` values are ready to go

Reply:
I removed any of the custom logic for months vs. quarters along with any tests